### PR TITLE
Migrate travis to github actions

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q             | A
| ------------- | ---
| Branch?       | "master" 
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT
| Doc PR        | reference to the documentation PR, if any

See CI in https://github.com/alexander-schranz/Routing/pull/1/checks

Requires:

 - [x] https://github.com/symfony-cmf/Testing/pull/208

### CODE REVIEW

**Review by dbu — COMMENTED** [review]
will need to restart after https://github.com/symfony-cmf/Testing/pull/208

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    push:
+        branches:
+            - "master"
+            - "[0-9]+.x"
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.[0-9]+.x"
+
+jobs:
+    test:
+        name: "PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }}"
```
Comment (dbu) [inline .github/workflows/test-application.yaml:14]:
```suggestion
        name: "PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }} ${{ matrix.dependencies }}"
```

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        runs-on: "ubuntu-20.04"
+        env:
+            SYMFONY_DEPRECATIONS_HELPER: weak
+
+        strategy:
+            fail-fast: false
+            matrix:
```
Comment (dbu) [inline .github/workflows/test-application.yaml:21]:
```suggestion
            matrix:
                symfony-version:
                    - '*'
```

**Comment by dbu** [comment]
hm, adding the star seems to have broken the build matrix. do you see whats wrong?

with the next push, testing should be available for php 8

**Comment by alexander-schranz** [comment]
> hm, adding the star seems to have broken the build matrix. do you see whats wrong?

I don't know tried the same and just the builds are then away. Looks for me more like a bug in github action itself. I think we need to remove the star.

**Comment by alexander-schranz** [comment]
Also tests green again: https://github.com/alexander-schranz/Routing/pull/1/checks

**Comment by dbu** [comment]
thanks!

**Pull request merged**